components/filter-field/src/filter-field.spec.ts

```diff
@@ -540,10 +540,51 @@ describe('DtFilterField', () => {
       tick(DT_FILTER_FIELD_TYPING_DEBOUNCE);
 
       fixture.detectChanges();
-      dispatchKeyboardEvent(inputEl, 'keyup', ENTER);
+      dispatchKeyboardEvent(inputEl, 'keydown', ENTER);
+
+      tick(DT_FILTER_FIELD_TYPING_DEBOUNCE);
+      fixture.detectChanges();
```

Comment:
shouldn't the fixture detect changes to be called before a tick? 
1. ChangeDetection
2. Zone